src/webchat-ui/components/presentational/chat-options/DeleteConversation.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Typography } from "@cognigy/chat-components";
+import Modal from "../../Modal/Modal";
+import { deletePrevConversation } from "../../../../webchat/store/previous-conversations/previous-conversations-reducer";
+import { getStorage } from "../../../../webchat/helper/storage";
+import { setShowChatOptionsScreen } from "../../../../webchat/store/ui/ui-reducer";
+import { StoreState } from "../../../../webchat/store/store";
+import styled from "@emotion/styled";
+import { IWebchatConfig } from "../../../../common/interfaces/webchat-config";
+import { clearMessages } from "../../../../webchat/store/messages/message-reducer";
+import Button from "../Button";
+import SecondaryButton from "../SecondaryButton";
+
+const Container = styled.div`
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+`;
+
+const DeleteButton = styled(Button)(({ theme }) => ({
+	background: theme.red,
+	color: theme.white,
```

Comment:
You can use the background-color and toggle between textDark & textLight, instead of setting the text to white. This will be helpful when the customer chooses 'white' or other light colors once we make the button background color customizable. (We have such an implementation for primary & secondary colors)